cura/ObjectManager.py

```diff
@@ -0,0 +1,87 @@
+from UM.Logger import Logger
```

Comment:
A list model is used as manager here as well, again a bit weird (see build plate model comments).